src/ai_company/templates/builtins/full_company.yaml

```
@@ -0,0 +1,68 @@
+  variables:
+    - name: "company_name"
+      description: "Name of your company"
+      default: "Acme Corp"
+    - name: "budget"
+      description: "Monthly budget in USD"
+      var_type: "float"
+      default: 200.0
+
+  company:
+    type: "full_company"
+    budget_monthly: {{ budget | default(200.0) }}
```

YAMLlint (1.38.0)

[error] too many spaces inside braces

(braces)